schema/project.js

```diff
@@ -1,3 +1,5 @@
+const { Op } = require('sequelize');
+const { merge } = require('lodash');
 const { STRING, ENUM, DATE, UUID, UUIDV1 } = require('sequelize');
```

Comment:
You're `require`-ing `sequelize` twice here. You should add `Op` to the other require call.